packages/squiggle-lang/src/rescript/Distributions/PointSetDist/PointSetDist_Scoring.res

```diff
@@ -0,0 +1,25 @@
+module KLDivergence = {
+  let logFn = Js.Math.log
+  let subtraction = (a, b) => Ok(a -. b)
+  let multiply = (a: float, b: float): result<float, Operation.Error.t> => Ok(a *. b)
+  let logScoreDirect = (a: float, b: float): result<float, Operation.Error.t> =>
+    if a == 0.0 {
+      Error(Operation.NegativeInfinityError)
+    } else if b == 0.0 {
+      Ok(b)
+    } else {
+      let quot = a /. b
+      quot < 0.0 ? Error(Operation.ComplexNumberError) : Ok(b *. logFn(quot))
+    }
+  let logScoreWithThreshold = (~eps: float, a: float, b: float): result<float, Operation.Error.t> =>
```

Comment:
There should be a good comment about what the threshold means and how it works.

Reply:
I talked more with Nuño and have a way of accomplishing without this hack.